.buildkite/scripts/steps/functional/security_solution.sh

```diff
@@ -5,12 +5,12 @@ set -euo pipefail
 source .buildkite/scripts/steps/functional/common.sh
 
 export JOB=kibana-security-solution-chrome
-export CLI_NUMBER=${CLI_NUMBER:-$((BUILDKITE_PARALLEL_JOB+1))}
-export CLI_COUNT=${CLI_COUNT:-$BUILDKITE_PARALLEL_JOB_COUNT}
+export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
+
+Xvfb :99 -screen 0 1600x1200x24 &
```

Comment:
this config is only run on the CI machine, I agree we could refactor this piece into the separate file @szwarckonrad 
the change is needed due to running multiple Cypress instances on the same machine, https://docs.cypress.io/guides/continuous-integration/introduction#Xvfb

Otherwise multiple Cypress runners tried to start their own Xvfb instance which cause CI issues